devops/scripts/benchmarks/requirements.txt

```diff
@@ -2,3 +2,4 @@ matplotlib==3.9.2
 mpld3==0.5.10
 dataclasses-json==0.6.7
 PyYAML==6.0.1
+Mako==1.3.0
```

Comment:
I think that all third-party requirements dependencies could be resolved by their `requirements.txt` files. Can the llvm bench scripts call `pip` with relevant `requirements.txt` files instead? With this, we wouldn't have to worry about possible dependencies updates, just install what third-parties need.

Reply:
> I think that all third-party requirements dependencies could be resolved by their requirements.txt files.

I couldn't find one for IGC. This didn't change though - mako just happened to be already installed in most scenarios because UR has it in its requirements.txt ;) I just stumbled on this when building on a fresh system.

Reply:
Ok if there is none for IGC.